Applications/WebPortal/WebApp/library/source/pages/account_directory_page/create_account_page.tsx

```diff
@@ -19,6 +21,9 @@ interface Properties {
   /** The status given back from the server on callback. */
   errorStatus?: string;
 
+  /** The model that provides the group suggestions. */
+  groupSuggestionModel: GroupSuggestionModel;
```

Comment:
This model will need to be loaded before the page gets displayed such as in ```componentWillMount```. Once the load is complete then you can render the page.

Reply:
`componentWillMount` is depreciated. Maybe inside the constructor?